src/Microsoft.TestPlatform.CommunicationUtilities/TestRequestSender.cs

```diff
@@ -622,8 +642,24 @@ private void OnTestRunAbort(IInternalTestRunEventsHandler testRunEventsHandler,
         SetOperationComplete();
 
         var reason = GetAbortErrorMessage(exception, getClientError);
-        EqtTrace.Error("TestRequestSender: Aborting test run because {0}", reason);
-        LogErrorMessage(string.Format(CultureInfo.CurrentCulture, CommonResources.AbortedTestRun, reason));
+        // If it isn't a generic reason then, log it dispite if blame enabled or no.
+        if (reason != "Test host process crashed")
+        {
+            EqtTrace.Error("TestRequestSender: Aborting test run because {0}", reason);
+            LogErrorMessage(string.Format(CultureInfo.CurrentCulture, CommonResources.AbortedTestRun, reason));
+        }
+        // If it's a generic reason if blame not enabled ask the user to enable it for more details. 
+        else if (!IsBlameEnabled())
+        {
+            EqtTrace.Error("TestRequestSender: Aborting test run");
+            LogErrorMessage(string.Format(CultureInfo.CurrentCulture, "The active test run was aborted. Use blame for more details."));
+        }
+        //If blame enabled and it's ageneric reason just add the log without reason.
+        else
+        {
+            EqtTrace.Error("TestRequestSender: Aborting test run");
+            LogErrorMessage(string.Format(CultureInfo.CurrentCulture, "The active test run was aborted."));
```

Comment:
Mention that blame was enabled and that they should look at the blame messages below?